src/frontend/scss/objects/_course_glimpses.scss

```
@@ -147,17 +159,56 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
 
   &__title {
     @include font-size($h6-font-size);
-    -webkit-box-orient: vertical;
-    -webkit-line-clamp: 3;
     color: r-theme-val(course-glimpse, title-color);
-    display: block;
-    display: -webkit-box;
     font-family: $r-font-family-montserrat;
     font-weight: $font-weight-boldest;
     flex: 1 0 1.3em * 3; // 3 lines;
     line-height: 1.3em;
     margin-bottom: 1rem;
+  }
+
+  &__link {
+    color: inherit;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 3;
+    display: block;
+    display: -webkit-box;
     overflow: hidden;
+
+    // we extend the clickable zone of the link,
+    // so that the clickable zone actually sticks to the edges of the card
+    padding: 1.7rem $course-glimpse-content-padding-sides 1rem;
+    margin: -1.7rem #{$course-glimpse-content-padding-sides * -1} -1rem;
```

The padding bottom could be increase to cover all the heading bounding box
```suggestion
    padding: 1.7rem $course-glimpse-content-padding-sides 1.7rem;
    margin: -1.7rem #{$course-glimpse-content-padding-sides * -1} -1.7rem;
```

---

If we do this and the title takes up 3 lines, the link goes above the organization text below so I'm not sure we should do this

---

But your comment helped me notice we can do better, now the link takes the whole title height as it should.

---

Ok, now should be good after moving the line-clamp stuff :pray: thanks